src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -69,12 +69,12 @@ public void periodic() {
 
 
         gyroValue = getGyro();
-        if (SmartDashboard.getBoolean("USE VISION",false)){
-            poseEstimator.updatePositionWithVis(gyroValue);
-        }else {
-            poseEstimator.updatePosition(gyroValue);
-        }
-
+//        if (SmartDashboard.getBoolean("USE VISION",true)){
```

Comment:
I don't understand this: Shouldn't we always update with Gyro?
Shouldn't we leave the boolean to control updates from vision?

Reply:
The original purpose of the USE VISION was to disable it until we were done testing. Because vision was working I got rid of the USE VISION. If we want to add it back however I think it would be fine (probably want to make it a constant instead of a shuffleboard value)

Reply:
Constant is fine. We should keep a feature flag for it somehow